backend/inspirehep/records/api/literature.py

```diff
@@ -237,7 +237,8 @@ def get_modified_references(self):
 
         return list(self.get_records_ids_by_pids(list(pids_changed)))
 
-    def update_authors_signature_blocks_and_uuids(self):
+    @staticmethod
+    def update_authors_signature_blocks_and_uuids(data):
```

Comment:
why ``static``?

Reply:
Because I don't have access to the record instance where I use it. An idea would also be to take it out of the LiteratureRecord class but I thought it is better to have it there since it is only used for Lit records